timemachine/ff/handlers/nonbonded.py

```diff
@@ -83,6 +87,12 @@ def oe_assign_charges(mol, charge_model=AM1BCCELF10):
     """assign partial charges, then premultiply by sqrt(ONE_4PI_EPS0)
     as an optimization"""
 
+    oemol = convert_to_oe(mol)
+
+    # recover the original atom order in case modified by SetCanonOrder(True)
```

Comment:
```suggestion
    # recover the original atom order in case ordering modified when generating conformers due to SetCanonOrder(True)
```

Reply:
Clarified in https://github.com/proteneer/timemachine/pull/1397/commits/88f8ff0e3dae7d61f4cd67286ae12b572d6b1d37